main-darwin.cpp

```diff
@@ -317,3 +317,38 @@ main (int argc,
     }
   }
 }
+
+/**
+ * Provide log message to filestream implementation for the engine.
+ */
+int jerry_port_logmsg (FILE* stream, const char* format, ...)
```

Comment:
We can put `jerry_port_logmsg`, `jerry_port_errormsg` and `jerry_port_putchar` to `main.h` until implementation of these function would become different.